aicspylibczi/CziFile.py

```diff
@@ -270,6 +270,39 @@ def read_subblock_metadata(self, **kwargs):
         m_index = self._get_m_index_from_kwargs(kwargs)
 
         return self.reader.read_meta_from_subblock(plane_constraints, m_index)
+    
```

Comment:
apparently there are spaces on this line (273) which is a minor listing error. 
if you could keep the newline but remove the spaces that would be awesome.